includes/image.inc

```diff
@@ -63,7 +63,7 @@ function template_preprocess_image_insert_image(&$vars) {
   // http://drupal.org/node/1923336
```

Comment:
Can you remove this comment too please? Don't think this is necessary anymore...